### PR TITLE
Misc Fixes while debugging another issue

### PR DESCRIPTION
**Pull request by srossross**
* Improved debug logging handlers
 * Added debug logging statements
 * Fix user-agent

@rafaelhbarros can you review?

### CODE REVIEW

File: binstar_client/utils/__init__.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ def parse_specs(spec):
 def load_token(url):
     tokenfile = join(dirs.user_data_dir, '%s.token' % quote_plus(url))
     if isfile(tokenfile):
+        log.debug("Found login token: %s", tokenfile)
```
Comment (rafaelhbarros) [inline binstar_client/utils/__init__.py]: is there any specific reason to not use `.format`?

**Comment by rafaelhbarros** [comment]
well, other than the error, it looks good.

**Comment by malev** [comment]
:+1: on `.format`

**Comment by srossross** [comment]
Now using `.format` what do you think? ready to merge?

**Comment by srossross** [comment]
oh, I see there are many other places I use the '%s', let me change thoes too.

**Comment by srossross** [comment]
Updated.

**Pull request merged**